microsite/data/plugins/glean.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: Glean
+author: Mark Shields
+authorUrl: https://github.com/beejiujitsu
+category: Search
+description: Index Backstage TechDocs-enabled Entities into Glean, the AI-powered work assistant
+documentation: https://github.com/beejiujitsu/wealthsimple-backstage-plugins/tree/master/plugins/glean-backend
```

Comment:
Would it be possible to share Wealthsimple's involvement with this plugin? It looks to be under your GitHub Org and NPM scope - `beejiujitsu` - but the repo is called `wealthsimple-backstage-plugins`.